lib/foreman/renderer/base_renderer.rb

```diff
@@ -16,11 +16,13 @@ def render
       def self.render(source, scope)
         result = new(source, scope).render
         digest = Digest::SHA256.hexdigest(result)
-        Foreman::Logging.blob("Unattended render of '#{source.name}' = '#{digest}'", result,
-          template_digest: digest,
-          template_name: source.name,
-          template_host_name: scope.host.try(:name),
-          template_host_id: scope.host.try(:id))
+        if !scope.preview? && source.template.is_a?(ReportTemplate)
```

Comment:
I might be a little overly strict here, but is there a way we can use inheritance for this?
Maybe `if !scope.previw && template.log_results?` ?